gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-oidc/src/main/java/io/gravitee/am/gateway/handler/oidc/service/idtoken/impl/IDTokenServiceImpl.java

```diff
@@ -133,7 +139,8 @@ public Single<String> create(OAuth2Request oAuth2Request, Client client, User us
                                 }
                                 return Single.just(signedIdToken);
                             });
-                });
+                })
+                .doOnError(error -> auditService.report(AuditBuilder.builder(ClientTokenAuditBuilder.class).tokenActor(client).tokenTarget(user).throwable(error)));
```

Comment:
**question**: Wondering if this one will create duplicate with the one made into the TokenServiceImpl https://github.com/gravitee-io/gravitee-access-management/pull/3285/files#diff-df69649536f25a3ab626f032370500d9ac5cfd8a0b21b46bd68434b6578874c2R161 :thinking:

Reply:
removed